src/test/java/org/opensearch/knn/index/query/KNNQueryBuilderTests.java

```diff
@@ -776,19 +777,23 @@ public void testDoToQuery_InvalidZeroByteVector() {
 
     public void testSerialization() throws Exception {
```

Comment:
nit: Does KNNQueryBuilderValidParamsTests and KNNQueryBuilderInvalidParamsTests need an update?

Reply:
Ill add some basic ones, but do most of testing in the Rescore validation tests